style.css

```diff
@@ -0,0 +1,196 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+.template5-main-mobile {
+  justify-items: center;
+  padding-top: 200px;
+  background: #191765;
+  width: 375px;
+  height: 5415px;
+}
+
+.app-bar {
+  position: absolute;
+  width: 200px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  line-height: 16px;
+  padding-bottom: 20px;
+}
+
+.fa-solid {
+  position: absolute;
+  font-size: 30px;
+  left: 301.33px;
+  top: 50px;
+  color: #c8c8d4;
+}
+
+.time {
+  position: absolute;
+  width: 60px;
+  height: 25px;
+  left: 7px;
+  top: 16px;
+}
+
+.name {
+  position: static;
+  padding-top: 50px;
+  padding-left: 10px;
+  color: #edecff;
+  text-decoration: none;
+}
+
+.ellipse1 {
+  position: absolute;
+  height: 160px;
+  width: 160px;
+  right: 321px;
+  left: -90px;
+  top: 97px;
+  bottom: 5158px;
+  background: #edecff;
+  filter: blur(50px);
+}
+
+.ellipse2 {
+  position: absolute;
+  height: 68px;
+  width: 100px;
+  left: 330px;
+  top: 495px;
+  bottom: 5158px;
+  background: #edecff;
+  filter: blur(50px);
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex: 1;
+}
+
+.primary-text {
+  font-size: 50px;
+  font-weight: 800;
+  font-style: normal;
+  line-height: 20px;
+  letter-spacing: 0.57px;
+  text-align: center;
+  padding-bottom: 40px;
+  color: #7d73fc;
+}
+
+.supporting-text {
+  width: 327px;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 4px;
+  letter-spacing: 0;
+  text-align: center;
+  padding-bottom: 40px;
+  color: #edecff;
+}
+
+.advert-text {
+  position: static;
+  width: 280px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #c8c8d4;
+}
+
```

Comment:
- [ ] Please, consider fixing this section. Inside of this `div` you have a `p` tag with a paragraph:
<img width="294" alt="Screen Shot 2022-02-15 at 18 20 34" src="https://user-images.githubusercontent.com/49064106/154080729-1a62efac-0b81-4481-acd3-b74b12f6e5f0.png">

Reply:
Noted